Makefile

```diff
@@ -0,0 +1,19 @@
+install_packages:
+	poetry install
+
+sort_imports:
+	isort .
+
+format:
+	black .
+
+lint:
+	flake8 .
+
+test:
+	pytest tests/
+
+precommit:
+	pre-commit install && pre-commit
+
+all: install_packages precommit test
```

Comment:
Consider making the `all` target the default one.

Reply:
done